docs/operator_parameters.md

```diff
@@ -0,0 +1,175 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+*CONFIG_MAP_NAME* environment variable. Variable names are delimiter-separated
+words.
+
+Available Configuration Parameters
+----------------------------------
+
+* **etcd_host**
+  Etcd host:port for Patroni. Not required when Patroni native Kubernetes
+  support is used.
+
+* **docker_image**
+  Spilo docker image for Postgres instances. For production, don't rely on a
+  default value, as it might be too old. Instead, build your own Spilo image
+  from the [github repository](github.com/zalando/spilo).
```

Comment:
imho worth mentioning the [official Spilo releases](https://github.com/zalando/spilo/releases)